extensions/ql-vscode/TELEMETRY.md

```diff
@@ -0,0 +1,45 @@
+# Telemetry in the CodeQL extension for VS Code
+
+If you specifically opt-in to permit GitHub to do so, GitHub will collect usage data and metrics for the purposes of helping the core developers to improve the CodeQL extension for VS Code. This data will not be shared with any parties outside of GitHub. IP addresses and installation IDs will be retained for a maximum of 30 days. Anonymous data will be retained for a maximum of 180 days.
```

Comment:
Well, I can't say "anonymous" because strictly speaking IP address and installation id is not anonymous. Even though we create the id, it is theoretically possible to map back to a single user. That's why I am separating "anonymous" data from IP address and installation id.

Reply:
Yeah, I think you're right. And I can't really think of a more accurate/appropriate word, so let's leave it as is.